core/src/test/java/org/frankframework/javascript/JavascriptEngineTest.java

```diff
@@ -27,22 +31,59 @@ public static void setup() {
 	void testBasicExecution(JavascriptSender.JavaScriptEngines engineWrapper) throws JavascriptException {
 		JavascriptEngine engine = engineWrapper.create();
 		engine.startRuntime();
-		engine.executeScript("function test(){ return 'Hello World!' }");
+		engine.executeScript("function test() { return 'Hello World!' }");
 		Object result = engine.executeFunction("test");
 		assertEquals("Hello World!", result.toString());
 		engine.closeRuntime();
 	}
 
+	// This test doesn't do much at the moment... We cannot capture the log output (yet).
```

Comment:
Klopt dit statement nog? Je had toch een LogHandler toegevoegd?

Reply:
Ja, maar de truffel logger is erg raar, heb hier 2uur in gestoken en was niets opgeschoten. In theorie zou alles moeten werken, er is een logger, maar ik krijg nooit log output...